internal/cmd/skupper/utils/encode.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"sigs.k8s.io/yaml"
+)
+
+func Encode(outputType string, resource interface{}) (string, error) {
+
+	var result []byte
+	var err error
+
+	switch outputType {
+	case "json":
+		result, err = json.Marshal(resource)
```

Comment:
We could use `MarshalIndent` here to have a pretty printed output, similar to the yaml version.

Reply:
done 17c770d23bb7501388e35c00df5d6d205c6254ad